easybuild/easyconfigs/c/CrayCCE/CrayCCE-19.06-clang.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'CrayToolchain'
+
+name = 'CrayCCE'
+version = '19.06-clang'
```

Comment:
@jgphpc, I think it is better to avoid the `-clang` on the version. Since this is the official version now. I would vote to have the toolchain without `-clang` and add `-classic` to any version that does not make use of Clang. This way we are compatible with Cray's own nomenclature.

What do you think?

Reply:
I said @jgphpc, but it was actually @lucamar.
What do you think @lucamar?